src/components/reviewMatches/DataGrid.tsx

```diff
@@ -0,0 +1,281 @@
+                  </Typography>
+                )
+              case 'Candidate':
+                if (params.row.searched) {
+                  return <Typography>Searched</Typography>
+                } else {
+                  return <Typography>Blocked</Typography>
+                }
+              default:
+                return <></>
+            }
+          }
+        }
+      }
+      if (fieldName === 'uid') {
+        return {
+          field: fieldName,
+          headerName: fieldLabel,
+          flex: 1,
+          renderCell: (params: GridRenderCellParams) => {
+            if (params.row.type === 'Current') {
+              return (
+                <Link href={`/patient-record/${params.row.uid}`}>
+                  {params.row.uid}
+                </Link>
+              )
+            }
+            return (
+              <Link href={`/golden-record/${params.row.uid}`}>
+                {params.row.uid}
+              </Link>
+            )
+          }
+        }
+      }
+      return {
+        field: fieldName,
+        headerName: fieldLabel,
+        flex: 1,
+        valueFormatter: (
+          params: GridValueFormatterParams<number | string | Date>
+        ) => formatValue(params.value),
+        cellClassName: (params: GridCellParams<string>) =>
+          getCellClassName(params, field, data[0])
+      }
+    }),
+    {
+      field: 'score',
+      headerName: 'Match',
+      type: 'number',
+      width: 100,
+      minWidth: 80,
+      align: 'center',
+      headerAlign: 'center'
+    },
+    {
+      field: 'actions',
+      headerName: 'Actions',
+      maxWidth: 180,
+      minWidth: 120,
+      flex: 1,
+      hide: hideAction,
+      align: 'center',
+      headerAlign: 'center',
+      sortable: false,
+      filterable: false,
+      valueGetter: (params: GridValueGetterParams) => ({
+        id: params.row.id,
+        patient: params.row.patient,
+        type: params.row.type
+      }),
+      renderCell: (params: GridRenderCellParams) => {
+        switch (params.row.type) {
+          case 'Current':
+            return (
+              <IconButton
+                aria-controls={isOpen ? 'basic-menu' : undefined}
+                aria-haspopup="true"
+                aria-expanded={isOpen ? 'true' : undefined}
+                onClick={open}
+                size="large"
+                edge="end"
+                disabled={true}
+              >
+                <MoreVertIcon />
+              </IconButton>
+            )
+          case 'Golden':
+            return (
+              <IconButton
+                aria-controls={isOpen ? 'basic-menu' : undefined}
+                aria-haspopup="true"
+                aria-expanded={isOpen ? 'true' : undefined}
+                onClick={open}
+                size="large"
+                edge="end"
+                disabled={true}
+              >
+                <MoreVertIcon />
+              </IconButton>
+            )
```

Comment:
Current and Golden appear to be the same.

```suggestion
          case 'Current':
          case 'Golden':
            return (
              <IconButton
                aria-controls={isOpen ? 'basic-menu' : undefined}
                aria-haspopup="true"
                aria-expanded={isOpen ? 'true' : undefined}
                onClick={open}
                size="large"
                edge="end"
                disabled={true}
              >
                <MoreVertIcon />
              </IconButton>
            )
```